### PR TITLE
Change [] to ()

### PR DESCRIPTION
**Pull request by ChristopherRogers1991**
Fixes https://github.com/MycroftAI/adapt/issues/91

### CODE REVIEW

**Comment by ChristopherRogers1991** [comment]
Apologies for the delay - I've added a simple unit test for `test_resolve_one_of`, which should fail on master but pass on this branch, and I've changed `.get` to `[]`.

**Comment by clusterfudge** [comment]
Sorry, it's going to take me a while to grok this test (need to revisit what are the inputs and outputs of this method). I think the more practical test is one that makes use of the "at least one" semantics of an intent definition, as opposed to asserting on the outputs of this deep-in-the-bowels method.

**Comment by ChristopherRogers1991** [comment]
@clusterfudge, I've added another test that should hopefully be more straightforward. The key is that a tag (`temperature' in the test) needs to be possible in multiple locations (passed in to multiple `one_of` calls), so that we have to advance to the second possible tag (update `last_end_index`) in one of those locations.

**Comment by ChristopherRogers1991** [comment]
Hey @clusterfudge,

Friendly ping. Let me know if you'd like me to make any other changes.

-Christopher

File: test/IntentTest.py [hunk 1]
```diff
@@ -70,6 +71,24 @@ def test_at_least_one(self):
             assert result_intent.get('PlayVerb') == 'play'
             assert result_intent.get('Radio Station') == "barenaked ladies"
 
+    def test_at_least_one_with_tag_in_multiple_slots(self):
```
Comment (clusterfudge) [inline test/IntentTest.py]: Ok, so to be clear, in the past this failed, and the behavior we want to assert on is that if the tag appears in multiple slots, the parse result with the largest phrase coverage wins. Sound right?

Comment (ChristopherRogers1991) [reply test/IntentTest.py]: Yes, that is correct.

**Comment by clusterfudge** [comment]
Cool, last request, please update the version in setup.py to 0.3.3

**Comment by ChristopherRogers1991** [comment]
@clusterfudge, done. The version is now 0.3.3.

**Comment by clusterfudge** [comment]
image

**Pull request merged**